cub/cub/device/dispatch/tuning/tuning_scan.cuh

```diff
@@ -293,7 +299,7 @@ struct policy_hub
 } // namespace detail
 
 // TODO(bgruber): deprecate this at some point when we have a better way to allow users to supply tunings
-template <typename AccumT, typename ScanOpT = ::cuda::std::plus<>>
-using DeviceScanPolicy = detail::scan::policy_hub<AccumT, ScanOpT>;
+template <typename AccumT, typename OffsetT, typename ScanOpT = ::cuda::std::plus<>>
+using DeviceScanPolicy = detail::scan::policy_hub<AccumT, OffsetT, ScanOpT>;
```

Comment:
Critical: This is a breaking change. We can work around this by supplying a dummy offset type that won't affect old tuning policies:
```suggestion
template <typename AccumT, typename ScanOpT = ::cuda::std::plus<>>
using DeviceScanPolicy = detail::scan::policy_hub<AccumT, int, ScanOpT>;
```